docs/overview.md

```diff
@@ -34,25 +33,24 @@ The main modules included in the `aerospike` package are:
   module:aerospike/lists|lists} and {@link module:aerospike/maps|maps} modules
   define the operations on scalar, list and map values that can be executed
   with the {@link Client#operate} command.
-* The info protocol provides access to configuration and statistics for the
-  Aerospike server. The {@link module:aerospike/info|info module} includes
-  utility functions for parsing the info data returned by the Aerospike cluster.
+* The {@link module:aerospike/info|info module} includes utility functions 
+  for parsing the info data returned by the Aerospike cluster. 
+  The info protocol provides access to configuration and statistics for the Aerospike server.
 
 ### Classes
 
 The main classes included in the `aerospike` module are:
 
-* {@link Client} - The main interface of the Aerospike client. Through the
-  Client class commands such as put, get or query can be sent to an Aerospike
-  database cluster.
+* {@link Client} - The main interface of the Aerospike client. Commands such as put, get or query can be sent to an Aerospike
+  database cluster using the Client class.
 * {@link Key} - Keys are used to uniquely identify a record in the Aerospike database.
 * {@link Record} - Records consists of one or more record "bins" (name-value
-  pairs) and meta-data, incl. time-to-live and generation; a
+  pairs) and meta-data (time-to-live and generation); a
   record is uniquely identified by it's key within a given namespace.
 * {@link Query} - The Query class can be used to perform value-based searches
```

Comment:
I think that in the Node.js client Scan should be deprecated and Query handles both primary index (PI) and secondary index (SI) queries now (that's what all language clients should have). If so, please be explicit about it in this description here

Reply:
The functionality may be available already, I will need to take a deeper dive.  The Scan and Query pages still include bits about predicate filtering and the query page doesn't mention searching primary indexes.  I will need to review and reformat those sections in a future release.